mteb/models/google_models.py

```diff
@@ -27,7 +50,8 @@ def __init__(
     def _embed(
         self,
         texts: list[str],
-        google_task_type: str | None = None,
+        google_task_type: str,
+        show_progress_bar: bool,
```

Comment:
Nit: Could we be more explicit?
```suggestion
        show_progress_bar: bool = False,
```